### PR TITLE
Code reformat in userinfo command

### PR DESCRIPTION
**Pull request by saiTama-max**
Moved `BADGES` dict to a yaml file in resources folder, and removed unnecessary code

### CODE REVIEW

**Comment by Xithrius** [comment]
Have you confirmed that this works?

**Comment by saiTama-max** [comment]
@Xithrius It is working now

**Comment by saiTama-max** [comment]
Currently its like this, 
image
image

File: bot/cogs/user.py [hunk 1]
```diff
@@ -39,31 +40,27 @@ async def get_member(ctx: Context, member: str) -> Member:
...
 
-        if months:
-            final.append(f'{months} months')
-
-        if years:
-            final.append(f'{years} years')
-        final = final[::-1]
+        for date in dates:
+            if date:
+                final.append(f"{date} {dates[date]}")
```
Comment (Xithrius) [inline bot/cogs/user.py:63]: You can definitely combine everything here into a single dictionary that gets looped over with `for k, v in dates.items()`.

In the current `dates` dictionary, I'd swap the keys and values so you can have a constant for the keys. All time deltas in the dictionary's new variable (not a variable, but a number that is suseptable to change) can have the absolute function used on them.

**Review by Xithrius — CHANGES_REQUESTED** [review]

**Review by Xithrius — APPROVED** [review]
LGTM

**Pull request merged**